pyxcp/tests/test_master.py

```diff
@@ -124,9 +124,7 @@ def testConnect2(self, mock_selector, mock_socket):
         mock_selector.return_value.select.side_effect = ms.select
 
         with Master(transport.Eth('localhost', loglevel="DEBUG")) as xm:
-            ms.push([
-                0x08, 0x00, 0x00, 0x00,
-                0xff, 0x1d, 0xc0, 0xff, 0xdc, 0x05, 0x01, 0x01])
+            ms.push(bytes.fromhex("08 00 00 00 FF 1D C0 FF DC 05 01 01"))
```

Comment:
bytearray objects accept list of integers for the extend method

Reply:
You're right.
It was more an experiment to see if the fromhex version is more readable?
What do you think? It's definitely shorter...

Reply:
It looks good. 

To make it even shorter https://github.com/christoph2/pyxcp/blob/master/pyxcp/tests/test_master.py#L16
```python
    def push(self, data):
        if isinstance(data, str):
            data = bytes.fromhex(data)
        self.data.extend(data)
```